### ISSUE TITLE
stmglossaries: Integrate option that allows the use of the abbreviation only for multiple occurences

### ISSUE THREAD

**Issue by raedma** [raedma/stmlatex#6]
Currently, the use of an acronym, e.g. `\gls{acr:DCB}`, produces double cantilever beam (DCB), which is totally fine and works as expected. However, for sake of clarity it might not be necessary to introduce the abbraviation (DCB) for the case that is not reused in a document and occurs only once.

Therefore, it would be nice to have an option, e.g. `abbrevationonlyformultipleoccurences=true`, or a separate style that allows this behavior and also prohibits the introduction in the list of abbrevations for single occurences.

See https://tex.stackexchange.com/questions/477581/glossaries-introduce-short-name-only-if-used-more-than-once-within-section-chap
